framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileResource.java

```diff
@@ -3,7 +3,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 
-/** Contract for sources of stub data. */
+/** Interface for annotation files (stub files). */
```

Comment:
Same comment as <https://github.com/jwaataja/checker-framework/pull/1>. Retain usage of word "source".